### PR TITLE
Add pttime.org support

### PR DESCRIPTION
**Pull request by tantao700**
Support a new PT site: https://www.pttime.org

jackett

### CODE REVIEW

File: src/Jackett.Common/Definitions/pttime.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+          args: "2006-01-02 15:04:05 -07:00"
+    date:
+      # time added
+      selector: td:nth-child(5):not(:has(span))
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "2006-01-0215:04:05 -07:00"
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/pttime.yml:122]: Is there a space missing between the date and time?
"2006-01-0215:04:05 -07:00" vs
"2006-01-02 15:04:05 -07:00"

**Review by garfield69 — APPROVED** [review]
Thank you.

File: src/Jackett.Common/Definitions/pttime.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        img.pro_50pctdown: 0.5
+        img.pro_50pctdown2up: 0.5
+        img.pro_30pctdown: 0.3
+        "*": 1
+    uploadvolumefactor:
+      case:
+        img.pro_50pctdown2up: 2
+        img.pro_free2up: 2
+        img.pro_2up: 2
+        "*": 1
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/pttime.yml:144]: Can you add the site's minimum ratio and minimum seeding time (if they exist)? e.g. https://github.com/Jackett/Jackett/commit/a016faba4faa3fcfa8458868925bb6dde5d81e4d

File: src/Jackett.Common/Definitions/pttime.yml (hunk 1 above)
Comment (garfield69) [reply src/Jackett.Common/Definitions/pttime.yml:122]: >Is there a space missing between the date and time?

no, when the torrentlist is in time-added mode,  there is usually a `<br>` in-between, which dateparse ignores, so this ok.

File: src/Jackett.Common/Definitions/pttime.yml (hunk 2 above)
Comment (tantao700) [reply src/Jackett.Common/Definitions/pttime.yml:144]: thanks your comments, 
according to the site's rules ,no H&R required,so no exist min seeding ratio and time.

File: src/Jackett.Common/Definitions/pttime.yml [hunk 3]
```diff
@@ -0,0 +1,148 @@
+---
+id: pttime
+name: PTTime
+description: "PTTime is a CHINESE Private Torrent Tracker for HD MOVIES / TV / GENERAL"
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/pttime.yml:4]:
```suggestion
description: "PTTime is a ratioless CHINESE Private Torrent Tracker for HD MOVIES / TV / GENERAL"
```

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]

**Review by ilike2burnthing — COMMENTED** [review]
Thanks, this will be in the next release in about 4.25hrs

**Pull request merged**